jobs/service-fabrik-broker/spec

```diff
@@ -38,6 +38,9 @@ properties:
   broker.enable_circuit_breaker:
     description: "Switch used to turn on/off circuit breaker"
     default: false
+  broker.feature.ServiceInstanceAutoUpdate:
```

Comment:
I understood the reason of camelCase here, but with all properties, is it looking odd?

Reply:
Yes. It does :-), but then if i make it camel case, it would look odd along with other job names in JS. So either ways it would look odd.

Reply:
True. Let's keep it as is then.